cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -10,35 +10,46 @@
 
    Create a hero molecule.
 
-   value:           An object with the following requirements.
+   value:               An object with the following requirements.
 
-   value.heading:   A string containing title of module.
+   value.is_overlay:    A boolean to change the hero to an overlay hero.
+   value.is_white_text: A boolean to change the text color to white.
```

Comment:
Locking down the approved background colors seems like a task for v2 when the designers have more bandwidth, and opening up the text colors seems unfriendly to content designers who shouldn't need to know what `#000` and `#fff` mean if those are going to be the only two options they need to use.